src/node_util.cc

```diff
@@ -174,6 +176,20 @@ void PromiseReject(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(ret.FromMaybe(false));
 }
 
+void SafeGetenvForJs(const FunctionCallbackInfo<Value>& args) {
+  Isolate* isolate = args.GetIsolate();
+  Local<String> envtag = args[0]->ToString();
+  String::Utf8Value strenvtag(envtag);
+  std::string text;
+  bool isEnvSafe = false;
+  isEnvSafe = SafeGetenv(*strenvtag, &text);
```

Comment:
Okay, doing it this way *does* make my TODO comment relevant – this is going to break on Windows if the environment variable is not pure ASCII.

Reply:
I think I'm not using it on any windows part of the code, right?